templates/unbound.conf.erb

```diff
@@ -1,162 +1,217 @@
 # Managed by Puppet
 #
 <%-
-  def bool2yesno (value)
-    return value ? 'yes' : 'no'
+  if @unbound_version
+    @_unbound_version = @unbound_version
```

Comment:
`_unbound_version`, doesn't need to be a class static variable, so you can leave the `@` out